tests/TaskLocalMixin.py

```diff
@@ -568,7 +568,7 @@ def testTaskAbortSelf(self):
 
         # retry
         task = task_self()
-        worker = task.shell("/bin/echo shouldnt see that")
+        worker = task.shell("/bin/echo shouldn't see that")
```

Comment:
we should just avoid the simple quote here, to make things safer

Reply:
ok, I propose to change *shouldn't* into *should not* in revised commit.